src/components/listitems/ListItemRadioWithAmount.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { StyleSheet, View } from "react-native";
 import RNReactNativeHapticFeedback from "react-native-haptic-feedback";
-import { IOColors, useIOTheme } from "../../core";
+import { IOColors, IOStyles, useIOTheme } from "../../core";
```

Comment:
Remove `IOStyles` because it's not necessary (see below)
```suggestion
import { IOColors, useIOTheme } from "../../core";
```